docs/doxygen/pages.dox

```diff
@@ -20,32 +20,37 @@ The configurations for memory estimation are defined <a href="https://github.com
 
 <table>
     <tr>
-        <td colspan="3"><b>Code Size of MQTT LTS rc1(example generated with GCC for ARM Cortex-M)</b></td>
+        <td colspan="3"><b>Code Size of coreMQTT (example generated with GCC for ARM Cortex-M)</b></td>
```

Comment:
```suggestion
        <td colspan="3"><b>Code Size of coreMQTT library files (example generated with GCC for ARM Cortex-M)</b></td>
```

Reply:
Would be good to hyperlink the ARM GCC toolchain

Reply:
Also, add a statement about these numbers being rounded up from their decimal values.